src/bert-classifier/Dockerfile.base

```diff
@@ -2,13 +2,16 @@
 
 FROM python:3.7-slim
 
+ARG CUDA_VERSION
+
 COPY requirements.txt /tmp/
 
 RUN mkdir -p /usr/share/man/man1/ && \
     apt-get update && apt-get install --no-install-recommends -y git default-jdk wget jq uuid-runtime && \
     apt-get clean -y -qq && apt-get autoremove -y -qq && \
     rm -rf /var/lib/apt/lists/* && \
     git clone https://github.com/pytorch/serve.git && \
-    cd serve && python ./ts_scripts/install_dependencies.py && \
+    cd serve && \
+    if [ -z "$CUDA_VERSION" ]; then python ts_scripts/install_dependencies.py; else python ts_scripts/install_dependencies.py --cuda "$CUDA_VERSION"; fi && \
```

Comment:
Delighted we used this `install_dependencies` script now!! Got cuda installation essentially for free!

https://github.com/pytorch/serve/blob/master/docker/Dockerfile.dev#L56